### DISCUSSION TITLE
Disconnect event is not triggered

### DISCUSSION THREAD

**Discussion by c-rick** [phantom/docs#322] [Help & Support] [answered]
### Summary

when i disconnect app, then disconnect event not triggered, but when i dispatch disconnect event , it triggered
<img width="332" alt="image" src="https://github.com/user-attachments/assets/f2e47883-9909-45ec-b2a5-7acb43f611f1">
```js
provider.request({ method: 'disconnect' }) // it trigged
provider.on('disconnect', () => {
  console.log('disconnected')
})
```

### Version

24.26.0

### Steps to Reproduce

provider.request({ method: 'disconnect' }) // it trigged
provider.on('disconnect', () => {
  console.log('disconnected')
})

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by c-rick** [comment]
<img width="1196" alt="image" src="https://github.com/user-attachments/assets/e9491ab2-46d7-4cbd-8f3b-31b18b57f5e5">
Is this your team? There is no Phantom Connect in the advertisement

**Reply by adamdelphantom** [reply]
@c-rick This is not our team. Please be careful w/ any links posted here. We have had a rash of bots posting scam links. Only contact support via settings from your phantom wallet.

**Comment by adamdelphantom** [comment]
@c-rick Am I understanding correctly that you are expecting a disconnect event to be triggered to your dapp when the user disconnects from settings?

**Reply by c-rick** [reply]
yep, But there is no trigger

**Reply by adamdelphantom** [reply, marked as answer]
@c-rick This event is not available via the provider.

**Reply by c-rick** [reply]
In addition to using a circular inspection to obtain a wallet status, do I have other ways to get this event?
